libs/designer/src/lib/ui/panel/connectionsPanel/createConnection/createConnectionWrapper.tsx

```diff
@@ -127,6 +128,12 @@ export const CreateConnectionWrapper = () => {
     dispatch(openPanel({ nodeId, panelMode }));
   }, [dispatch, referencePanelMode, nodeIds]);
 
+  const queryClient = useQueryClient();
+  const useUpdateNewConnection = useMutation(async (newConnection: Connection) => {
```

Comment:
```suggestion
  const updateNewConnection = useMutation(async (newConnection: Connection) => {
```

use** shouldn't be how a function name starts unless it's a hook, or else it gets treated different by eslint